src/Microsoft.VisualStudio.ProjectSystem.Managed/Threading/Tasks/CancellationSeries.cs

```diff
@@ -66,7 +66,9 @@ public CancellationSeries(CancellationToken token = default)
         /// <exception cref="ObjectDisposedException">This object has been disposed.</exception>
         public CancellationToken CreateNext(CancellationToken token = default)
         {
+#pragma warning disable RS0030 // Do not used banned APIs (cannot use alternative API here)
```

Comment:
Just out of curiosity, why is it not possible to use `CancellationTokenExtensions.CombineWith` here?

Reply:
It returns a struct and we need a reference type for the interlocked operation.

Reply:
Could you declare it as a nullable of that struct type, since nullable is a reference type? (Keep in mind, I don't fully understand the context here, so my question might not make any sense.)

Reply:
Nullable structs are still value types, so that wouldn't help here unfortunately. There's probably a way to change the design here but it'd increase the complexity a fair bit I suspect, or introduce blocking synchronization.

Reply:
Doy. Not sure why I thought `Nullable<T>` was a reference type. My bad.